content/practices/valid-tags.txt

```diff
@@ -0,0 +1,13 @@
+### special tags ###
+all methods
+draft
+example tag
+
+### content tags ###
+Scoping
+Kickoff
+Discovery
+Framing
+Inception
+Transition
+Modernization
```

Comment:
Not a huge deal but I wonder if we maybe shouldn't keep this text file in the content folder? It will be published and addressable. Maybe better in a README (and eventually maybe a GitHub action)?

Reply:
That works for for me. This was a carry-over from the old site generator.

Reply:
Removing, will deal with later.